samples/RXPTest/src/Tests/ViewBasicTest.tsx

```diff
@@ -106,6 +110,59 @@ class BasicView extends RX.Component<RX.CommonProps, RX.Stateless> {
                           'I have differing border radii' }
                     </RX.Text>
                 </RX.View>
+
+                <RX.View 
+                    style={ _styles.explainTextContainer } 
+                    key={ 'explanation2' }
+                >
+                    <RX.Text style={ _styles.explainText }>
+                        { 'Should be read by screen reader as \"Alpha Beta Gamma Delta\" ' + 
+                          'and screen reader should not be able to get to individual texts.' }
+                    </RX.Text>
+                </RX.View>
+                <RX.View style={ _styles.labelContainer }>
+                    <RX.View 
+                        style={ _styles.labelContainer } 
+                        importantForAccessibility={ RX.Types.ImportantForAccessibility.Yes }
+                    >
+                        <RX.Text style={ _styles.labelText }>
```

Comment:
Should we verify the behavior of explicitly setting `importantForAccessibility` to `Auto`?

Reply:
Good idea to test Auto too! Will do.

---
In reply to: [161925938](https://github.com/Microsoft/reactxp/pull/431#discussion_r161925938) [](ancestors = 161925938)